ts/features/itwallet/common/hooks/useItwPendingReviewRequest.ts

```diff
@@ -3,21 +3,24 @@ import { useIsFocused } from "@react-navigation/native";
 import { useDispatch, useSelector } from "react-redux";
 import { itwIsPendingReviewSelector } from "../store/selectors/preferences";
 import { itwSetReviewPending } from "../store/actions/preferences";
-import { requestAppReview } from "../../../../utils/storeReview";
+import { useAppReviewRequest } from "../../../appReviews/hooks/useAppReviewRequest";
 
 /**
  * Hook to monitor isPendingReview state and request an app review if needed.
  * If isPendingReview is true, request an app review and then set isPendingReview to false.
  */
 export const useItwPendingReviewRequest = () => {
+  const { requestFeedback, bottomSheet } = useAppReviewRequest("itw");
```

Comment:
What do you think about using a more expressive name for the bottom sheet used for feedback, similar to other parts? For example, `appFeedbackBottomSheet`.